app/models/donation_site.rb

```diff
@@ -19,6 +22,8 @@ class DonationSite < ApplicationRecord
   belongs_to :organization
 
   validates :name, :address, :organization, presence: true
+  validates :contact_name, length: {minimum: 3}, allow_blank: true
+  validates :email, format: {with: URI::MailTo::EMAIL_REGEXP, message: "is not a valid email format"}, allow_blank: true
```

Comment:
Good job re-using an existing email regexp rather than rolling your own. Also when the migration runs, all of the email columns will be blank so it is important that you have the `allow_blank: true` 👍